From f7fa48c4c75a1e748dc5071e709c0b62ff739eaa Mon Sep 17 00:00:00 2001
From: "jan.nijtmans" <nijtmans@users.sourceforge.net>
Date: Mon, 9 Dec 2019 10:02:20 +0000
Subject: [PATCH] Fix [abd4abedd2]: Failed to build tk 8.6.10 with cross
 compile

Patch taken from upstream to fix cross-compilation. To be removed on next Tcl release.

---
 compat/strtol.c  | 2 +-
 compat/strtoul.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/compat/strtol.c b/compat/strtol.c
index b7f69196ce2..811006a64c3 100644
--- a/compat/strtol.c
+++ b/compat/strtol.c
@@ -53,7 +53,7 @@ strtol(
      */
 
     p = string;
-    while (TclIsSpaceProc(*p)) {
+    while (isspace(UCHAR(*p))) {
 	p += 1;
     }
 
diff --git a/compat/strtoul.c b/compat/strtoul.c
index e37eb05f823..15587f1da1a 100644
--- a/compat/strtoul.c
+++ b/compat/strtoul.c
@@ -74,7 +74,7 @@ strtoul(
      */
 
     p = string;
-    while (TclIsSpaceProc(*p)) {
+    while (isspace(UCHAR(*p))) {
 	p += 1;
     }
     if (*p == '-') {
